### PR TITLE
Add classification template

### PR DESCRIPTION
**Pull request by KumoLiu**
Part of tutorial#1456

### Description
Add a classification template

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: models/classification_template/configs/inference.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+# transform to apply to data from network to be suitable for validation
+postprocessing:
+  _target_: Compose
+  transforms:
+  - _target_: Activationsd
+    keys: '@pred'
+    softmax: true
+  - _target_: AsDiscreted
+    keys: ['@pred', '@label']
+    argmax: [true, false]
```
Comment (ericspod) [inline models/classification_template/configs/inference.yaml:76]:
```suggestion
  - _target_: AsDiscreted
    keys: '@pred'
    argmax: true
```

I don't think we want to apply anything to the label here?

File: models/classification_template/configs/train.yaml [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+# transform to apply to data from network to be suitable for validation
+postprocessing:
+  _target_: Compose
+  transforms:
+  - _target_: Activationsd
+    keys: '@pred'
+    softmax: true
+  - _target_: AsDiscreted
+    keys: ['@pred', '@label']
+    argmax: [true, false]
```
Comment (ericspod) [inline models/classification_template/configs/train.yaml:141]: Same thing I suggested above.

**Comment by ericspod** [comment]
Hi @KumoLiu looks good! I had a few points about how transforms are being applied but it works for you so it's just a best practice thing. One other suggestion is to have a `common.yaml` file with all the common definitions in it (eg. `network_def`), then include that when doing things like training. I hadn't done that for the segmentation template because I wanted to maintain compatibility with the current version of MONAILabel. Here we don't need to worry about that so we can change things now to have a common file or leave it for a future refactor.

File: models/classification_template/configs/inference.yaml (hunk 1 above)
Comment (KumoLiu) [reply models/classification_template/configs/inference.yaml:76]: I added this one here to one hot label that may used in `ConfusionMetrix`.

**Comment by KumoLiu** [comment]
Hi @ericspod, I didn't get the point of why we need the `common.yaml`. I see you created an issue [here](https://github.com/Project-MONAI/MONAILabel/issues/1549) and also mentioned it in the dev meeting. But seems we don't need to specify the common field in multiple configs, we only need to add what we want to override in the `evaluate.yaml` or `inference.yaml` or other configs.

**Comment by KumoLiu** [comment]
After reading again the configs, I think I get the points. It makes sense if we add a `common.yaml` but every time we may need to specify multiple configs due to the existence of `common.yaml`.
Perhaps we could refactor it later when we reach a consensus. What do you think?

**Comment by ericspod** [comment]
I'm good to refactor later, it's something we will likely do for both template bundles. I had recommended having a common config file in [this bundle tutorial](https://github.com/Project-MONAI/tutorials/blob/main/bundle/03_mednist_classification_v2.ipynb) but it's not necessary.

**Review by ericspod — APPROVED** [review]
LGTM!
